SpaDES4Dummies.Rmd

```diff
@@ -180,9 +200,11 @@ abundanceInit <- function(sim) {
 }
 ```
 
+Since we are only interested in simulating and plotting species abundances, we can remove unnecessary events from the script. Only 3 events were kept: the initialisation, an abundance simulation event and a plotting event.
```

Comment:
Could we move this bit to before we talk about the initialization? Maybe put it 
 ####**2. Event functions**{#anchor2}
[HERE]
 #####*Initialisation function*